velox/exec/Spill.h

```diff
@@ -62,6 +64,7 @@ class SpillMergeStream : public MergeStream {
   /// batch, in which case the caller must call copy out current batch data if
   /// required before calling pop().
   vector_size_t currentIndex(bool* isLastRow = nullptr) const {
+    VELOX_CHECK(!closed_);
```

Comment:
ditto

Reply:
Same reason. reset the `rowVector_` in close function.